websites/P/PERPHeads/presence.ts

```diff
@@ -133,10 +127,7 @@ presence.on("UpdateData", async () => {
       title = document.querySelector(
         "div.p-body > div.p-body-inner > div.p-body-header > div.p-title > h1"
       );
-      search = document.querySelector(
-        "div.p-body > div.p-body-inner > div.p-body-header > div.p-title > h1"
-      );
-      presenceData.state  = title.innerText.replace(search.innerText, "").replace("»", "");
+      presenceData.state  = title.innerText;
```

Comment:
2 spaces instead of one

Reply:
Fixed all these now and bumped version :)